index.html

```diff
@@ -7,6 +7,6 @@
     <title>Document</title>
 </head>
 <body>
-    <h1>HELLO MICROVERSE</h1>
+    <h1>HELLO MICROVERSE!</h1>
```

Comment:
- [ ] kindly consider giving a class to your  header and use it  to style your h1 which will help your project   to meet the below requirement 
- > A .css file that includes one class with styles for your header (make it in your favorite color).